.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+Pleas give a clear and concise description of what the bug is here.
+
+**To Reproduce**
+If you can describe the steps needed to reproduce here that will help us squash the bug faster.  If you can reproduce the bug using simple code, please put that code here.
+
+**Cirq version**
+Please describe what version of Cirq you are using.  If you installed cirq via pip you can get the version by running `pip show cirq`.
```

Comment:
Optional: Something like `python -c 'import cirq; print(cirq.__version__)'` has the benefit of working regardless of how cirq was obtained.

Reply:
done